crates/bifrost/src/loglet/loglet_tests.rs

```diff
@@ -387,11 +387,26 @@ pub async fn single_loglet_readstream_with_trims(
         loglet.append(format!("record{}", i).into()).await?;
     }
 
+    // When reading record 8, it's acceptable to observe the record, or the trim gap. Both are
+    // acceptable because replicated loglet read stream's readhead cannot be completely disabled.
```

Comment:
For my own understanding: If we could completely disable the read ahead, wouldn't it still be possible to either read a record or a trim gap? Trimming requires reaching write-quorum. Let's assume that the remaining trim tasks fail. Then depending whether we read from a log server that was trimmed or not, we might either see a trim gap or the record?

Reply:
It's possible, but remember that we execute GetTrimPoint on readstream start which will try to get quorum (best-effort still) trim-point, and this reduces that chances of this happening (if any of the trimmed nodes is still alive)